controllers/db_participation_operations.py

```diff
@@ -88,3 +90,15 @@ def verify_if_arrived(self, event_id, user_id):
         result = {'enrolled': bool(joined), 'arrived': bool(appeared)}
         return to_json(result)
 
+
+# external method to call user service
+    def give_points(self, event_id, user_id):
+        event = self.events_col.find_one({"_id": ObjectId(event_id)})
+        points = event['points']
+        success = give_influence_points(points, user_id)
```

Comment:
Btw, I have updated the user-service to also include actual user_id, which is created upon registration of the user... 
Was bad not to in the first place - sorry. 
Now either of the ways should work - if you want to get the user by `username` or by `user_id`
From the readme: https://github.com/Volunteero/user-service/blob/master/README.md
> PUT: ../altar/v1/users/find
purpose: find a user by request
body in: (request)  
`{
	"username":"dafoe5"  
}`
or
`{
	"user_id":"dafoe5"  
}`
note: if the input body contains both user_id and username, the preference is for the latter

Sorry for the inconvenience...